alibi/explainers/cfproto.py

```diff
@@ -178,7 +178,11 @@ def __init__(self,
         self.max_iterations = max_iterations
         self.c_init = c_init
         self.c_steps = c_steps
-        self.feature_range = feature_range
+        self.feature_range = list(feature_range)
+        for _ in range(2):
```

Comment:
At first I though iterating over only the first 2 items of the tuple is a clever way to get rid of any extra items the user may have passed, but that is actually not the case as we have already assigned to `self.feature_range`, so it will always be of the length the user passed (which could be longer than 2).

Not saying we have to validate explicitly here as we don't do much validation elsewhere (also assuming implicitly below that `feature_range` has at least 2 items), but more of a note for the future.

Reply:
Yes, but we never check/validate that it is the case.